### ISSUE TITLE
Feature Request: share a password from one user to an other

### ISSUE THREAD

**Issue by Anoerak** [strongbox-password-safe/Strongbox#644]
Hi, 

Could you implement a feature that would allow one Strongbox user to share a password (all entries) with another one user as you can do with Bitwarden.
So this way, when you want to share a password with let say your partner, and you don't use a common database, you can "send"/"share" an entry with him/her without having to send it through insecure channels like messages or airdrop ...
It would be encrypted all along.

https://bitwarden.com/blog/whats-the-right-way-to-share-passwords/

Thank you :)

**Comment by SVilgelm** [comment]
+1 for this feature

**Comment by unicorn855** [comment]
Isn't this already available? Provided both databases are on the same device (can Air Drop the database itself first if needed), you just need to unlock the source database and then long-press on the entry you want to share, then choose 'export', and then choose/unlock the target database. After that, close the source database and unlock the target database again to make sure it's synchronised correctly, then just Air Drop the other person's database back to their device (and hope it synchronises with their version of the database)

I hope that helps :-)

**Comment by SVilgelm** [comment]
@unicorn855 what version do you use? I found the export action on the whole database, but not on a particular entry:
![IMG_4329](https://user-images.githubusercontent.com/523825/192586594-facd5456-8a58-4991-b185-17e6b411ab16.jpeg)

I don't need to share the db itself, but I need to be able to share/send an entry.

**Comment by unicorn855** [comment]
Interesting! I am running verspinn 1.56.3 Pro on iOS 16.0.2 (current) and the export option shows under 'duplicate' for me. Is the screenshot you took from the MacOS version? I have been using the Pro version for iOS ever since and I have no Mac to test, but it is my understanding that both the versions for iOS and MacOS offer the same functionalities? Could it be a pro thing? Are you also using the app on mobile?

**Comment by SVilgelm** [comment]
I use Strongbox with lifetime license, not a PRO. I really didn't know about the PRO version at all :(
the version is same 1.56.3 on IOS 16.0.2

same on macOS version, there is no Export/Share action for an Item

**Comment by strongbox-mark** [comment]
HI @SVilgelm - If you have a LIfetime license that is a PRO version :)

You won't see the Export option if you have only a single database. You can only export to other databases and so the option is hidden for users with only one.

Hope that makes sense :) Thanks for your support!

**Comment by strongbox-mark** [comment]
HI @SVilgelm - If you have a LIfetime license that is a PRO version :)

You won't see the Export option if you have only a single database. You can only export to other databases and so the option is hidden for users with only one.

Hope that makes sense :) Thanks for your support!

**Comment by SVilgelm** [comment]
Oh, I see, thank you! Yeah, I have just a single database. 
So the original feature request is still valid.

**Comment by strongbox-mark** [comment]
OK, yes, gotcha. I think what @unicorn855 was trying to say was there's a way to do this using multiple databases...

However, could you describe in a bit more detail how you would see this working, or how you've seen it done in other apps?

**Comment by unicorn855** [comment]
From what I can see on the website, there is only a free and a pro version, so it shouldn't matter how you paid (lifetime should be pro just like the subscription-based pro version). Did you take the screenshot you shared here from the iOS version? Perhaps it's a big of some kind that prevents you from having the option?
Maybe @mmcguill  has more ideas about what might be up here?
Does the app say its pro (when you tap the cogwheel icon, it should tell you under the 'pro status' section)?

**Comment by SVilgelm** [comment]
To me, as a user, I need an ability to send an entry/item to someone, like my wife or a child, using airdrop.

How I see this: long press on an item -> share -> (standard share menu, like airdrop, message, email, etc..) -> select a recipient -> maybe enter a pin or password to protect the item
on another device with installed Strongbox: incoming request -> open strong box -> enter pin or password of imported item -> select a database -> select a folder -> save

**Comment by strongbox-mark** [comment]
I see, have you seen this done in another app that we could compare/investigate?

**Comment by SVilgelm** [comment]
yeah, on iPhone, open `Settings -> Passwords -> long press on a record -> Share with AirDrop...`

![IMG_4333](https://user-images.githubusercontent.com/523825/192677937-393118ae-04bd-472b-87a7-7e49935f5de0.png)

**Comment by SVilgelm** [comment]
I think the AirDrop is the most secure option in such case, I'm ok with AirDrop only, no messages, no emails and etc...

**Comment by Anoerak** [comment]
You can see something similar on Bitwarden 😁
